app/src/main/java/com/github/onlynotesswent/ui/search/Search.kt

```diff
@@ -278,29 +338,87 @@ fun SearchScreen(
       }
 }
 
+/**
+ * Displays a deck item in the search screen.
+ *
+ * @param deck The deck to display.
+ * @param author The author of the deck.
+ * @param onClick The action to perform when the deck is clicked.
+ */
+@Composable
+fun DeckSearchItem(deck: Deck, author: String, onClick: () -> Unit) {
```

Comment:
Not needed for this pr. But making this a common deck card like for note would have allowed us to reuse this for the overview of the deck.

Reply:
yes I can do that